### ISSUE TITLE
Impossible reproduce your results in the paper

### ISSUE THREAD

**Issue by arith3** [birdortyedi/instagram-filter-removal-pytorch#2]
1. No test datasets (train datasets have 16 classes)
2. Codes that cannot be trained (learning rate not match)
3. Different model structure compared with your paper architecture

How did you guys come up with the results presented in the workshop?
